config/crds/starlingx_v1_host.yaml

```diff
@@ -436,6 +436,86 @@ spec:
                         - port
                         type: object
                       type: array
+                    vf:
+                      description: VF defines the list of SR-IOV VF interfaces to
+                        be configured on a host.
+                      items:
+                        properties:
+                          class:
+                            description: Class defines the intended usage of this
+                              interface by the system.
+                            enum:
+                            - platform
+                            - data
+                            - pci-sriov
+                            - pci-passthrough
+                            - none
+                            type: string
+                          dataNetworks:
+                            description: DataNetworks defines the list of data networks
+                              to be configured against this interface.
+                            items:
+                              maxLength: 255
+                              pattern: ^[a-zA-Z0-9\-_]+$
+                              type: string
+                            maxLength: 255
+                            pattern: ^[a-zA-Z0-9\-_]+$
+                            type: array
+                          lower:
+                            description: Lower defines the interface name over which
+                              this VF interface is to be configured.
+                            pattern: ^[a-zA-Z0-9\-_\.]+$
+                            type: string
+                          mtu:
+                            description: MTU defines the maximum transmit unit for
+                              this interface.
+                            format: int64
+                            maximum: 9216
+                            minimum: 576
+                            type: integer
+                          name:
+                            description: SystemName defines the name of the interface
```

Comment:
Not that it matters but SystemName is not the field here.

Reply:
Thanks Al.  I've refreshed the pull request. Seems it was an existing typo in hostprofile_types.go ->CommonInterfaceInfo that affected the description of all interface types.  There's a similar issue for VolumeGroupInfo and EthernetPortInfo.  I can fix those up in a followup PR, as they are not directly associated with this one.